source/core/Math.cpp

```diff
@@ -275,8 +275,20 @@ DECLARE_SCALE2X_INTN_HELPER(Single)
 // Comparison
 #define DECLARE_VIREO_COMPARISON_PRIMITIVES_BASE(TYPE) \
     DECLARE_VIREO_PRIMITIVE3(IsLT##TYPE, TYPE, TYPE, Boolean, (_Param(2) = _Param(0) <  _Param(1))) \
+    DECLARE_VIREO_PRIMITIVE3(IsLTSort##TYPE, TYPE, TYPE, Boolean, \
+        if (::isnan((double)_Param(1))) { \
```

Comment:
Please add (if they don't exist already) gvi and via tests using all the comparison operators comparing NaN's and Inf's and check those tests in along with this changeset.